### PR TITLE
[13.0][MIG] Migrated hotel_housekeeping.

### PR DESCRIPTION
**Pull request by dhruvitaSerpentCS**

### CODE REVIEW

File: hotel_housekeeping/report/housekeeping_report.py [hunk 1]
```diff
@@ -9,7 +9,9 @@
 
 
 class ActivityReport(models.AbstractModel):
+
     _name = "report.hotel_housekeeping.housekeeping_report"
+    _description = "report.hotel_housekeeping.housekeeping_report"
```
Comment (Trivedi-Vacha-SerpentCS) [inline hotel_housekeeping/report/housekeeping_report.py]: the description should be like "Housekeeping Report"

File: hotel_housekeeping/wizard/housekeeping_wizard.py [hunk 2]
```diff
@@ -5,12 +5,13 @@
 
 class HotelHousekeepingWizard(models.TransientModel):
     _name = "hotel.housekeeping.wizard"
+    _description = "hotel.housekeeping.wizard"
```
Comment (Trivedi-Vacha-SerpentCS) [inline hotel_housekeeping/wizard/housekeeping_wizard.py]: Same as above, improve _description

**Review by Trivedi-Vacha-SerpentCS — CHANGES_REQUESTED** [review]

**Review by vimalpatelserpentcs — APPROVED** [review]

File: hotel_housekeeping/wizard/housekeeping_wizard.py (hunk 2 above)
Comment (JayVora-SerpentCS) [reply hotel_housekeeping/wizard/housekeeping_wizard.py]: addressed it ?

**Review by Trivedi-Vacha-SerpentCS — APPROVED** [review]

File: hotel_housekeeping/wizard/housekeeping_wizard.py (hunk 2 above)
Comment (dhruvitaSerpentCS) [reply hotel_housekeeping/wizard/housekeeping_wizard.py]: Yes already changed it in the last commit.

**Pull request merged**